### PR TITLE
LPS-104826 Fix duplicate tooltips

### PR DESCRIPTION
**Pull request by wincent**
Rebase of https://github.com/wincent/liferay-portal/pull/118 with some updates on top.

### CODE REVIEW

**Comment by wincent** [comment]
ci:test:sf

**Comment by wincent** [comment]
ci:test:stable

**Comment by wincent** [comment]
ci:test:relevant

File: modules/apps/frontend-js/frontend-js-tooltip-support-web/src/main/resources/META-INF/resources/index.js [hunk 1]
```diff
@@ -210,7 +210,7 @@ const TooltipProvider = () => {
 
 			saveTitle(state.target);
 		}
-	}, [state]);
+	}, [state.target]);
```
Comment (wincent) [inline modules/apps/frontend-js/frontend-js-tooltip-support-web/src/main/resources/META-INF/resources/index.js]: @jbalsas: Does this change look safe to you?

Comment (jbalsas) [reply modules/apps/frontend-js/frontend-js-tooltip-support-web/src/main/resources/META-INF/resources/index.js]: As long as `state.target` changes if `state` changes properly in the `reducer`, it should be.

I think we mostly nuke `target` in the transitions where it's not needed, so it should potentially be safe, but I have a bad feeling about this :D

Comment (wincent) [reply modules/apps/frontend-js/frontend-js-tooltip-support-web/src/main/resources/META-INF/resources/index.js]: > I have a bad feeling about this

bad-feeling

I'll take that as a green light then! In my manual testing everything *seems* to work.

#NoLogicChanges

**Comment by wincent** [comment]
CI seems stalled, so forwarding here to try again: https://github.com/brianchandotcom/liferay-portal/pull/83324

**Pull request closed without merging**